packages/utils/types.ts

```diff
@@ -10,10 +10,12 @@ export {
   isPromise,
   isSymbol,
 } from '@vue/shared'
-export { isBoolean, isNumber } from '@vueuse/core'
+export { isClient, isIOS } from '@vueuse/core'
```

Comment:
`types.ts` is for type checking. They should go to `browser.ts`.

Reply:
Done.  IIs it better to extract and manage other methods imported from `vueuse`?